sdk/openai/Azure.AI.OpenAI/src/Custom/AzureCognitiveSearchChatExtensionConfiguration.Serialization.cs

```diff
@@ -17,17 +17,18 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
             writer.WritePropertyName("type"u8);
             writer.WriteStringValue(Type.ToString());
 
```

Comment:
The customization for `ImageSize` could be removed as well.

Reply:
You're right. Thank you for catching this!